server/pulp/server/db/connection.py

```diff
@@ -91,12 +98,26 @@ def initialize(name=None, seeds=None, max_pool_size=None, replica_set=None, max_
                                                                 itertools.repeat(32))
 
         if seeds != '':
-            seeds_list = seeds.split(',')
             if len(seeds_list) > 1 and not replica_set:
                 raise PulpCodedException(error_code=error_codes.PLP0041)
             while True:
                 _CONNECTION = _connect_to_one_of_seeds(connection_kwargs, seeds_list, name)
                 if _CONNECTION:
+                    db_min_version_tuple = tuple(MONGO_MINIMUM_VERSION.split("."))
+                    wc_min_version_tuple = tuple(MONGO_WRITE_CONCERN_VERSION.split("."))
```

Comment:
Why not just create these constants as tuples above so you can drop these two lines?